src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Types.cs

```diff
@@ -90,22 +90,23 @@ private void VisitArrayType(IArrayTypeSymbol symbol, TypeSymbolWithAnnotations t
             }
         }
 
-        private void AddNullableAnnotations(TypeSymbolWithAnnotations typeOpt)
+        private void AddNullableAnnotations(TypeSymbolWithAnnotations? typeOpt)
```

Comment:
>TypeSymbolWithAnnotations? [](start = 44, length = 26)

Would it make sense to use a non-nullable TSWA, and use the default value for "nothing"?